keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -800,9 +803,9 @@ Graduation to Beta
 
 Graduation to GA:
 
-- Address any issues uncovered in alpha/beta
+- Add documentation for running as a non-SYSTEM/admin account to k8s.io
+- Update documention on how volume mounts are set up for `hostProcess` containers on k8s.io
```

Comment:
should we also have a requirement to be using this new volume mounts in capz or a similar project?